.github/dependabot.yml

```
@@ -9,3 +9,8 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Let’s keep this weekly. Monthly could be a long time and would rather know early if somethings gonna cause issues.

Any less than weekly is a lot of noise for little extra value but monthly feels too long to me.

```suggestion
      interval: "weekly"
```